src/test/java/bio/terra/workspace/service/trace/StackdriverTraceTest.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.workspace.service.trace;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import bio.terra.workspace.app.Main;
+import bio.terra.workspace.app.configuration.StackdriverConfiguration;
+import io.opencensus.common.Scope;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@Tag("unit")
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = Main.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+class StackdriverTraceTest {
+
+  @Autowired
+  private StackdriverTrace trace;
+
+  @Test
+  void createScope() {
+    String span1Name = "span1";
+    Scope scope = trace.scope(span1Name);
+
+  }
+
+  @Test
+  void annotate() {}
+}
```

Comment:
Yeah...this file is sad. I'm not sure how to test the StackdriverTrace class without somehow creating a mock Stackdriver. It may just be too thin for tests to actually create any value here

Reply:
I agree that that it's pretty thin. `annotate()` doesn't do anything except wrap the call; I don't think that's worth fighting against.

For the future, or for this PR if you feel like tackling it: you *could* write a custom exporter for unit tests, and use that instead of the stackdriver exporter: https://opencensus.io/exporters/custom-exporter/java/trace/. That would allow you to write a test that says "when I call a given API, do I properly capture the traces I expected to capture?"